### PR TITLE
feat: Add an option to print LLVM using the generated MLIR

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**
Changes in this PR:
- Simplify the code for handling the MLIR
- Add a new member function `llvm_str` in MLIRModule
- Add a new option `--show-mlir-to-llvm`

### CODE REVIEW

File: src/bin/lfortran.cpp [hunk 1]
```diff
@@ -2232,6 +2212,7 @@ int main_app(int argc, char *argv[]) {
     app.add_option("--skip-pass", skip_pass, "Skip an ASR pass in default pipeline");
     app.add_flag("--show-llvm", show_llvm, "Show LLVM IR for the given file and exit");
     app.add_flag("--show-mlir", show_mlir, "Show MLIR for the given file and exit");
+    app.add_flag("--show-mlir-to-llvm", show_mlir_to_llvm, "Show LLVM IR after translating from MLIR for the given file and exit");
```
[the remark this answers is not in the document]
Comment (kmr-srbh) [reply src/bin/lfortran.cpp:2215]: `--show-llvm-from-mlir` can be a good option. It puts the phrase `show-llvm` in the front so we know it's LLVM which will be printed. If we later decide to translate to LLVM from something else, like a certain ASR pass, then we can go with the `--show-llvm-mlir` option. For the ASR variant, we could say `--show-llvm-asr="asr_pass"`.

Comment (kmr-srbh) [inline src/bin/lfortran.cpp:2215]:
```suggestion
    app.add_flag("--show-mlir-to-llvm", show_mlir_to_llvm, "Show LLVM IR translated from MLIR for the given file and exit");
```

Does this look good to you?

Comment (Thirumalai-Shaktivel) [reply src/bin/lfortran.cpp:2215]: Yes.

**Comment by Thirumalai-Shaktivel** [comment]
@kmr-srbh, thanks for the suggestions.
I will push the required changes tomorrow.

**Pull request merged**